back/app/policies/project_policy.rb

```diff
@@ -118,11 +122,13 @@ def reorder?
   end
 
   def destroy?
-    active? && admin?
+    return false unless active?
+
+    admin? || (active_moderator? && record.never_published?)
   end
 
   def copy?
-    create?
+    active_moderator?
```

Comment:
What is the difference between `can_moderate` and `active_moderator`?

Reply:
Good point. They are a bit redundant. Main difference I guess is that `#active_moderator?` checks if the user is active. I'd rather not rework that now as `#active_moderator?` is a public method that is used in other places in the code base.

I could potentially rewrite it using `#can_moderate?` but to little benefit.
```ruby
def active_moderator?
  active? || can_moderate?
end
```